assets/js/plugin-check-admin.js

```diff
@@ -73,9 +154,12 @@
 		const pluginCheckData = new FormData();
 		pluginCheckData.append( 'nonce', pluginCheck.nonce );
 		pluginCheckData.append( 'plugin', data.plugin );
-		pluginCheckData.append( 'checks', data.checks );
 		pluginCheckData.append( 'action', 'plugin_check_run_checks' );
 
+		for (var i = 0; i < data.checks.length; i++) {
+			pluginCheckData.append('checks[]', data.checks[i]);
+		}
```

Comment:
Please pay attention to correct spacing per WP coding standards. We may want to set us JS linting via ESLint for this repo (maybe open an issue for that?).
```suggestion
		for ( var i = 0; i < data.checks.length; i++ ) {
			pluginCheckData.append( 'checks[]', data.checks[ i ] );
		}
```

Reply:
Thanks @felixarntz updated and created a new issue here https://github.com/10up/plugin-check/issues/102